### PR TITLE
Parameters added to values.yaml

### PR DESCRIPTION
**Pull request by Maladie**
**Description**

Changes proposed in this pull request:

- Parameters added to values.yaml
- Docs and READMEs updated

**Related issue(s)**

#4396

### CODE REVIEW

File: components/application-operator/README.md [hunk 1]
```diff
@@ -21,11 +21,16 @@ Application Operator performs different operations as a result of the following
  - **domainName** - Domain name of the cluster. The default domain name is `kyma.local`.
  - **namespace** - Namespace where the Application charts will be deployed. The default namespace is `kyma-integration`.
  - **tillerUrl** - Tiller release server URL. The default is `tiller-deploy.kube-system.svc.cluster.local:44134`.
+ - **helmTLSKeyFile** - Path to TLS key used for Tiller communication. The default is `/etc/certs/tls.key`.
```
Comment (klaudiagrz) [inline components/application-operator/README.md]:
```suggestion
 - **helmTLSKeyFile** - Path to the TLS key used for Tiller communication. The default is `/etc/certs/tls.key`.
```

File: components/application-operator/README.md [hunk 2]
```diff
@@ -21,11 +21,16 @@ Application Operator performs different operations as a result of the following
...
+ - **helmTLSCertificateFile** - Path to TLS certificate used for Tiller communication. The default is `/etc/certs/tls.crt`.
```
Comment (klaudiagrz) [inline components/application-operator/README.md]:
```suggestion
 - **helmTLSCertificateFile** - Path to the TLS certificate used for Tiller communication. The default is `/etc/certs/tls.crt`.
```

File: docs/application-connector/05-04-connector-service.md [hunk 3]
```diff
@@ -21,6 +21,7 @@ This table lists the configurable parameters, their descriptions, and default va
 | **deployment.args.appValidityTime** | Specifies a period of time during which certificates that the service issues for an Application are valid. It is provided in days. | `92d` |
 | **deployment.args.runtimeValidityTime** | Specifies a period of time during which certificates that the service issues for a runtime are valid. It is provided in days. | `92d` |
 | **deployment.args.central** | Determines whether the Connector Service works in the central mode. | `false` |
+| **deployment.args.requestLogging** | Enables logging of incoming requests | `false ` |
```
Comment (klaudiagrz) [inline docs/application-connector/05-04-connector-service.md]:
```suggestion
| **deployment.args.requestLogging** | Enables logging of incoming requests. | `false ` |
```

File: components/application-operator/README.md (hunk 1 above)
Comment (akgalwas) [reply components/application-operator/README.md]: Done

File: components/application-operator/README.md (hunk 2 above)
Comment (akgalwas) [reply components/application-operator/README.md]: Done

File: docs/application-connector/05-04-connector-service.md (hunk 3 above)
Comment (akgalwas) [reply docs/application-connector/05-04-connector-service.md]: Done

**Pull request merged**